### ISSUE TITLE
Subtle: Inline styling

### ISSUE THREAD

**Issue by blerude** [AriCima/prj-rev-bwfs-dasmoto#1]
https://github.com/AriCima/prj-rev-bwfs-dasmoto/blob/master/Dasmotos/index.html#L19

While entering your stylings directly into the 'style' attribute works, it's better to just create a class in your CSS and apply stylings that way, especially since you mostly apply stylings via classes in the rest of the document.

**Comment by AriCima** [comment]
Yes that was an issue, but I don't know which tag should I use in the HTML file to then apply the class selector in CSS. Could you please let me know which tag is the suitable to modify just part of one paragraph?

**Comment by blerude** [comment]
I recommend creating a class, like “price” because you’re describing the
prices, and applying it to all three instances.

**Comment by AriCima** [comment]
Yes, I got that, but my problem is in the html file. Which tag should I use? Because that is a part of a paragraph <p> and if I apply the class as <p class=“price”> it will be apply to the whole text.

What is your suggestion?

Thanks again

Ariel

**Comment by blerude** [comment]
You have the actual text that you're styling inside a `<span>`. You can keep that as is, but just change the style="..." to class="name". So line 19 might look like this: 

`<p>Made of the highest quality oak, Hacksaw brushes are known for their weight and ability to hold paint in large amounts. Available in different sizes. <span class="price"> Starting at $3.00 / brush.</span></p>`

Does that help?

**Comment by AriCima** [comment]
Allright ! Thanks a lot for your answer!

Cheers!

Ariel
